packages/scene-composer/src/store/slices/ViewOptionStateSlice.ts

```diff
@@ -31,4 +30,10 @@ export const createViewOptionStateSlice = (
       draft.lastOperation = 'setTagSettings';
     });
   },
+  setMatterportViewer: (enabled: boolean) => {
```

Comment:
something like `setMatterportViewerEnabled`? setMatterportViewer sounds like it's setting a viewer..

Reply:
Done

Reply:
Done